src/test/spec/unified_runner/test_event.rs

```diff
@@ -0,0 +1,93 @@
+use crate::{
+    bson::Document,
+    test::{CommandEvent, Matchable},
+};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub enum TestEvent {
```

Comment:
Ditto above comment -- this file is a copy of `test_event.rs` with the only change being that test events are camelCased in the new runner format rather than snake_cased.